app/Template/category/create.php

```diff
@@ -8,7 +8,7 @@
     <?= $this->form->text('name', $values, $errors, array('autofocus', 'required')) ?>
 
     <?= $this->form->label(t('Color'), 'color_id') ?>
-    <?= $this->form->select('color_id', array('' => t('No color')) + $colors, $values, $errors, array(), 'color-picker') ?>
```

Comment:
You have to keep the default "No color", otherwise this is going to brake the existing behaviour for existing categories without colour.

Reply:
What do you mean by the existing behavior? The CSS is still well rendered with the modification.